### PR TITLE
EUREKA-65: add script to migrate data from mod-consortia

### PR DESCRIPTION
**Pull request by yaroslav-kiriak**
## Purpose
Add script to migrate data from mod-consortia

## Approach
Example of parsed script with specific tenant name value:

image

## TODOS and Open Questions

- [ ] Check logging

## Pre-Merge Checklist:

Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added, or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do Rally stories exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail? Which endpoints/schemas changed, etc.
  - [ ] Do they have all the appropriate links to blocked/related issues?
- Are the Rally stories under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally, all the PRs involved in breaking changes would be merged on the same day to avoid breaking the folio-testing
environment. Communication is paramount if that is to be achieved, especially as the number of inter-module and
inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the
responsibility of the PR assignee.

### CODE REVIEW

**Review by mykolam-epm — APPROVED** [review]

File: src/main/java/org/folio/consortia/service/FolioTenantService.java [hunk 1]
```diff
@@ -26,7 +30,8 @@
 @Primary
 public class FolioTenantService extends TenantService {
 
-  private static final String EXIST_SQL = "SELECT EXISTS(SELECT 1 FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = ?)";
+  private static final String EXIST_SQL =
```
Comment (OleksiiKuzminov) [inline src/main/java/org/folio/consortia/service/FolioTenantService.java:33]: Where is this variable used?

File: src/main/java/org/folio/consortia/service/FolioTenantService.java [hunk 2]
```diff
@@ -62,6 +68,37 @@ public FolioTenantService(JdbcTemplate jdbcTemplate,
     this.customFieldsRetryProperties = customFieldsRetryProperties;
   }
 
+  /*
+   * Because of the liquibase.Scope implementation for the SpringLiquibase it is not possible to run several
+   * SpringLiquibase executions simultaneously. That is why this method must be synchronized.
+   */
+  @Override
+  public synchronized void createOrUpdateTenant(TenantAttributes tenantAttributes) {
+    beforeTenantUpdate(tenantAttributes);
```
Comment (OleksiiKuzminov) [inline src/main/java/org/folio/consortia/service/FolioTenantService.java:77]: Is it necessary to make this method call here?

File: src/test/java/org/folio/consortia/base/BaseRepositoryTest.java [hunk 3]
```diff
@@ -15,7 +15,7 @@
 import org.springframework.test.annotation.DirtiesContext;
 
 @Log4j2
-@DataJpaTest
+@DataJpaTest(properties = "spring.liquibase.parameters.tenantname=consortium")
```
Comment (OleksiiKuzminov) [inline src/test/java/org/folio/consortia/base/BaseRepositoryTest.java:18]: Where I can find the migration test?

File: src/main/java/org/folio/consortia/service/FolioTenantService.java (hunk 1 above)
Comment (yaroslav-kiriak) [reply src/main/java/org/folio/consortia/service/FolioTenantService.java:33]: on line 122

File: src/main/java/org/folio/consortia/service/FolioTenantService.java (hunk 2 above)
Comment (yaroslav-kiriak) [reply src/main/java/org/folio/consortia/service/FolioTenantService.java:77]: it is empty, also `beforeLiquibaseUpdate` and `afterLiquibaseUpdate`, thus I am gonna get rid of them

**Review by OleksiiKuzminov — APPROVED** [review]

**Pull request merged**